src/order-management/portfolioUtils.ts

```
@@ -1,4 +1,9 @@
-import { Portfolio, Balance } from '../types';
+import { Portfolio, Balance, Position } from '../types';
+import { Decimal } from 'decimal.js';
+
+export type HistoricalReturns = {
+  [symbol: string]: number[];
+};
 import { Decimal } from 'decimal.js';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Decimal'. Consider to delete it or rename it.

'Decimal' is defined here:


(lint/suspicious/noRedeclare)